### ISSUE TITLE
Jaeger not showing any services

### ISSUE THREAD

**Issue by kahootali** [istio/istio#2222]
I am using Istio 0.2.12, kubernetes 1.8. Minikube 0.23 .
I have followed [this tutorial](https://istio.io/docs/tasks/telemetry/distributed-tracing.html) to integrate Jaeger for tracing but Jaeger doesn't show any services whereas Zipkin is running fine. Am I missing something?
This is the what I get for jaeger services.

```
istio-system   jaeger-agent           ClusterIP   None         <none>        5775/UDP,6831/UDP,6832/UDP                               3d
istio-system   jaeger-collector       ClusterIP   10.0.0.236   <none>        14267/TCP,14268/TCP,9411/TCP                             3d
istio-system   jaeger-query           NodePort    10.0.0.34    <none>        80:30175/TCP                                             3d
```
I have even tried on Azure Cluster but having same response.

![image](https://user-images.githubusercontent.com/13013437/34107887-b65db8c8-e41f-11e7-8cc9-1bcbf3a75666.png)

**Comment by pavolloffay** [comment]
@kahootali hi,

How are you reporting data to Jaeger? Are you using Zipkin tracers e.g. Brave or sleuth? 

Could you please add log from `kubectl get all -n istio-system`

**Comment by kahootali** [comment]
@pavolloffay hi,
I am not sure as I did not configure Brave or sleuth manually. I just followed the steps in the tutorial.
I just applied 
`kubectl apply -n istio-system -f https://raw.githubusercontent.com/jaegertracing/jaeger-kubernetes/master/all-in-one/jaeger-all-in-one-template.yml
`
and then accessing the dashboard.
Below is the output for  `kubectl get all -n istio-system`

![log get all](https://user-images.githubusercontent.com/13013437/34142846-5a37d254-e4ab-11e7-94eb-ca9f843fcf6b.PNG)
